src/main/java/com/service/QuoteService.java

```diff
@@ -1,60 +1,68 @@
-package com.service;
-
-import java.util.List;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
-
-import com.dao.QuoteRepository;
-import com.model.Quote;
-
-
-/**
- * 
- * QuoteService class for accessing the dao layer.
- * 
- * @author Neutralizer
- *
- */
-@Service
-public class QuoteService {
-
-	@Autowired
-    private QuoteRepository quoteRepository;
-
-
-	/**
-	 * Return all quotes from repo.
-	 * @return All quotes
-	 */
-    public List<Quote> getAllQuotes() {
-        return quoteRepository.getAllQuotes();
-    }
-
-    /**
-     * 
+package com.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.dao.QuoteRepository;
+import com.model.Quote;
+
+
+/**
+ * 
+ * QuoteService class for accessing the dao layer.
+ * 
+ * @author Neutralizer
+ *
+ */
+@Service
+public class QuoteService {
+
+	@Autowired
+    private QuoteRepository quoteRepository;
```

Comment:
Prefer constructor injection instead of field injection

Reply:
Done in neutralizer/jpa-data-adding branch